src/backend/distributed/deparser/citus_ruleutils.c

```diff
@@ -390,6 +392,54 @@ pg_get_tableschemadef_string(Oid tableRelationId, IncludeSequenceDefaults
 			}
 #endif
 
+			if (attributeForm->attidentity && includeIdentityDefaults)
+			{
+				Oid seqOid = getIdentitySequence(RelationGetRelid(relation),
+												 attributeForm->attnum, true);
+				Assert(seqOid != InvalidOid);
```

Comment:
Before going into `getIdentitySequence` function you can define `bool missing_ok = false` and pass that to the function. This way, you can omit the assertion as well.